content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -162,6 +162,17 @@ gets scheduled onto one node and then cannot run there, which is bad because
 such a pending Pod also blocks all other resources like RAM or CPU that were
 set aside for it.
 
+{{< warning >}}
+
+Scheduling of pods which use ResourceClaims is going to be slower because of
+the additional communication that is required. Beware that this may also impact
+pods that don't use ResourceClaims because only one pod at a time gets
+scheduled, blocking API calls are made while handling a pod with
+ResourceClaims, and thus scheduling the next pod gets delayed.
+
+{{< /warning >}}
```

Comment:
Try this:
```suggestion
{{< note >}}
Scheduling of pods that use ResourceClaims is going to be slower because of
the additional communication that is required. Beware that using ResourceClaims
in your cluster may also impact scheduling of pods that don't use ResourceClaims.
This impact happens because the kube-scheduler schedules one pod at a time;
blocking API calls are made while handling a pod with ResourceClaims, and thus
scheduling for other Pods in the scheduling queue is also delayed.
{{< /note >}}
```

Don't use a `warning`; it's not the best fit for this level of advice.

---

Once DRA is beta, we might want to add a one-line mention in https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/ about potential sources of scheduling delays, and _link_ to this page for more detail of how DRA can contribute there.

Reply:
Changed to `note` and created [an issue](https://github.com/kubernetes/kubernetes/issues/121869) to track updating `pod-lifecycle`.

Reply:
(the suggestion also proposed a wording tweak, though it's not immediately obvious from the diff)